src/@next/Banner/Banner.tsx

```diff
@@ -21,7 +22,8 @@ export type BannerProps = {
   secondaryAction?: React.ReactNode;
   dismissable?: boolean;
   onDismiss?: () => void;
-};
+  type?: 'static' | 'fixed' | 'fixed-noicon';
```

Comment:
I wasn't sure if this is the best thing... so based on the specification:
- type can be static (default)
- type can be fixed, and **only** for fixed type  there is an option to set the icon as optional

For now this is my approach

Reply:
If I read the jira ticket here: https://glints.atlassian.net/browse/SS-1057, it seems like change behavior from previously static to fixed, do we still need static? Also what's the `fixed-noicon` for? 🤔

Reply:
Figma link: https://www.figma.com/file/87fhOdF8vk1Uiy20bCqS85/Sirius-Web---UI-Kit-3.0-%5BQ4-2022%5D?type=design&node-id=4633-49953&t=mHqK1ifLLMEo7v3B-0

I think they still need the static banner and was just requesting a new variant (fixed) banner. In the Figma above, on left bottom it is written there:
> Only the fixed banner can set the icon as optional

So I assume for static banner, there should always be an icon (yes, the icon prop is optional, but there is a default icon value); while for this one, I assume they want an option to not have the icon completely

Reply:
Perhaps after this, I'll also be creating the visual test cases for the banner with two actions too

Reply:
@michac789 , but why do we need that as an option tho? From the `bannerStyle` file they have the same style.

Reply:
I'm more incline to have a prop called `showIcon` or something

Reply:
It is used in `Banner.tsx` here only: (line 87-91)

```
          type === 'fixed' && (
            <StyledBannerContentContainer data-type={type}>
              <Icon name={iconNameValue} />
            </StyledBannerContentContainer>
          )
```


At first I also wanted to create a boolean prop `showIcon`, but the Figma says only fixed banner can has no icon, and the original (static) type should always has an icon.

It is different from checking whether the `icon` prop is given or not. By default, an icon will be given based on the `status` prop if `icon` prop is not given.

Reply:
I think we can make `showIcon` default to `true` here as that's a default behavior and shouldn't be  rendered when it's  `false`. If user set no icon for `static` then I think they should be consciously doing that. WDYT?

Reply:
sounds good!